drivers/sensor/lsm6dsl/Kconfig

```diff
@@ -8,41 +8,93 @@
 #
 
 menuconfig LSM6DSL
-	bool "LSM6DSL I2C accelerometer and gyroscope Chip"
-	depends on I2C
+	bool "LSM6DSL I2C/SPI accelerometer and gyroscope Chip"
+	depends on SENSOR && (I2C || SPI)
 	default n
 	help
-	  Enable driver for LSM6DSL I2C-based accelerometer and gyroscope
+	  Enable driver for LSM6DSL accelerometer and gyroscope
 	  sensor.
 
-if !HAS_DTS_I2C_DEVICE
```

Comment:
If not using dst SPI bindings yet, at least you should not remove it for I2C.

Reply:
Yes, probably I misunderstood the purpose of this flag. I leave it.